spec/jobs/searchgov_url_fetcher_job.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe SearchgovUrlFetcherJob do
+  subject(:perform) { SearchgovUrlFetcherJob.perform_now(args[:searchgov_url]) }
```

Comment:
Because you aren't using [keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments) for this job, you can just say:
`subject(:perform) { SearchgovUrlFetcherJob.perform_now(searchgov_url) }`
...and remove the `args` variable.

Reply:
Did you mean to remove the `args` variable from the `SearchgovUrlFetcherJob.perform_now(args[:searchgov_url])` line, from this spec altogether, or both (I've so far only removed it from the `*.perform_now` statement)?